reposerver/repository/repository.go

```diff
@@ -1185,6 +1185,7 @@ func mergeSourceParameters(source *v1alpha1.ApplicationSource, path, appName str
 	merged.Path = source.Path
 	merged.RepoURL = source.RepoURL
 	merged.TargetRevision = source.TargetRevision
+	merged.FetchSubmodules = source.FetchSubmodules
```

Comment:
@crenshaw-dev not sure if this is needed or not but adding just in case.

Reply:
Yep, definitely needed.  :-)